pytorch_lightning/trainer/trainer.py

```diff
@@ -1101,15 +1101,19 @@ def run_pretrain_routine(self, model: LightningModule):
 
     def test(
             self,
+            *args,
```

Comment:
this is quite a breaking API change, I would rather have only `**kwargs` and rename it as `loading_kwargs`
cc: @williamFalcon @PyTorchLightning/core-contributors

Reply:
awesome pr! let's think about this a bit more carefully? mind giving us a few days to recover from the release? haha @yukw777

Reply:
Haha no worries guys, 0.8 was a huge release!! Sleep on this and let me know what you think.